### DISCUSSION TITLE
@pinia/nuxt: Cannot find module 'pinia/dist/pinia.mjs'

### DISCUSSION THREAD

**Discussion by niklasfjeldberg** [nuxt/nuxt#16500] [Questions]
I am unable to use `@pinia/nuxt` in my project, and I am unable to recreate the issue. Have anyone experienced this problem before, who could help me out?

I get the following error when I run `npm run dev` with `@pinia/nuxt` in my modules in `nuxt.config.ts`.
```
ERROR  Cannot restart nuxt:  Cannot find module 'pinia/dist/pinia.mjs'
Require stack: - C:\Users\XXXX\Prosjekter\projectX\index.js
```
The error shows up without me actively using the module in my project, all I do is to add it to my modules.

```
- Operating System: `Windows_NT`
- Node Version:     `v16.15.0`
- Nuxt Version:     `3.0.0-rc.8`
- Package Manager:  `npm@8.11.0`
- Builder:          `vite`
- User Config:      `ssr`, `modules`, `strapi`, `googleFonts`, `publicRuntimeConfig`, `css`, `vite`
- Runtime Modules:  `@vueuse/nuxt@9.1.0`, `@nuxtjs/strapi@1.5.0`, `@nuxtjs/google-fonts@3.0.0-0`, `@pinia/nuxt@0.4.1`
- Build Modules:    `-`
```

**Comment by danielrjames** [comment]
I ran into this after upgrading to rc8. I solved it by upgrading @pinia/nuxt and pinia to the latest release.

**Reply by niklasfjeldberg** [reply]
Thanks for the reply. I belive `@pinia/nuxt` is on the latest release, which should be `0.4.1` according to the [package page](https://www.npmjs.com/package/@pinia/nuxt).

**Reply by mikespinks0401** [reply]
@niklasfjeldberg Did you figure it out? I'm currently at the same exact spot

**Reply by niklasfjeldberg** [reply]
@mikespinks0401 no, unfortunately.

**Reply by Lumethys** [reply]
hi, i read somewhere and manage to fix this by adding an alias in nuxt.config:

```
alias: {
        pinia: '/node_modules/@pinia/nuxt/node_modules/pinia/dist/pinia.mjs',
    },
```

**Comment by danielroe** [comment]
If you provide a reproduction, I'll happily take a look.

**Comment by niklasfjeldberg** [comment]
I belive adding the package "pinia" to dev solved my problem.

**Comment by yuanquan8** [comment]
Here's how I did it
``` ts [nuxt.config.ts]
const lifecycle = process.env.npm_lifecycle_event;
export default defineNuxtConfig({
  ...
  build: {
        transpile:lifecycle === 'build' || lifecycle === 'generate' ? ['pinia'] : []
  }
  ...
})
```
